BSP/Simulator_V2/Src/Simulator.c

```diff
@@ -40,6 +42,25 @@ static time_t startTime;
 #define JSON_PATH STRINGIFY(SIMULATOR_JSON_PATH) 
 #endif
 
+static uint16_t DUMMY_VOLTAGES[NUM_BATTERY_MODULES] = {
+    42, 42, 42, 42, 42,
+    42, 42, 42, 42, 42,
+    42, 42, 42, 42, 42,
+    42, 42, 42, 42, 42,
+    42, 42, 42, 42, 42,
+    42, 42, 42, 42, 42,
+    42
```

Comment:
If the json file does not specify the value of the voltages, make them a value that doesn't immediately trip the BPS.

Reply:
Got it

Reply:
I think we would want to trip the BPS though if the test tries to measure the voltages, but we haven't provided voltage data, right?

Reply:
I think @ClarkPoon said that these voltage values should be ok, but I modified the temperature values to be a safe value of 30C.